.github/workflows/ci.yml

```diff
@@ -63,8 +63,8 @@ jobs:
                     use_slepc: false
         env:
             # make sure to pin this in tox.ini as well
-            PC_VERSION: 3.15.4  # PETSc version
-            SC_VERSION: 3.15.2  # SLEPc version
+            PC_VERSION: 3.18  # PETSc version
```

Comment:
This should also include the minor version as `3.18.0`, that's the reason why the CI script fails.